.github/workflows/uffizzi-build.yml

```diff
@@ -33,11 +31,10 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           file: ./uffizzi/Dockerfile
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
```

Comment:
Did caching not work for this project?

Reply:
Nope - it works strangely - when caching is enabled - the pipeline fails with a strange error.